src/components/authentication/index.tsx

```diff
@@ -1,3 +1,23 @@
+import { useState } from 'react'
+import Login from './Login'
+import Register from './Register'
+
+
 export default function Index() {
-  return <div>index</div>;
+  const [isLogin, setIsLogin] = useState(true)
```

Comment:
Declare appropriate types for all state variables
for instance: `const [isLogin, setIsLogin] = useState<boolean>(true)`

Reply:
resolved